app/models/katello/erratum.rb

```diff
@@ -46,10 +46,8 @@ def self.repository_association_class
     end
 
     def self.applicable_to_hosts(hosts)
-      self.joins(:content_facet_errata).joins(:content_facets).
-         where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).
-         select("DISTINCT ON (#{self.table_name}.updated, #{self.table_name}.id) #{self.table_name}.*").
-         order("#{self.table_name}.updated desc, #{self.table_name}.id")
+      self.joins(:content_facets).
+        where("#{Katello::Host::ContentFacet.table_name}.host_id" => hosts).uniq
```

Comment:
This query was "correct", but what happened was another user of the same query was appending more stuff to it. So what ended up happening was this query had an `ORDER BY` and then someone else came in and added another `ORDER BY`, causing :broken_heart:.

I thought about it some today but I can only think of ways to test for this with integration tests like bats. One example would be for bats to query the errata API at test time, which would have found this issue.